app/main.js

```diff
@@ -57,24 +57,10 @@ $(document).on("click", "a:not([data-bypass])", function (evt) {
   }
 });
 
-function getReducers (r) {
-
-  if (!r.length) {
-    return function () {};
-  }
-
-  return FauxtonAPI.reducers.reduce((el, acc) => {
-    acc[el] = el;
-    return acc;
-  }, {});
-}
-
-const reducer = getReducers(FauxtonAPI.reducers);
-
 const middlewares = [thunk];
 
 const store = createStore(
-  reducer,
+  combineReducers(FauxtonAPI.reducers),
```

Comment:
Instead of having a single reducer we use redux combineReducer so we can name space the different reducers
http://redux.js.org/docs/api/combineReducers.html